test/quantization/eager/test_quantize_eager_ptq.py

```diff
@@ -74,6 +75,73 @@
 import numpy as np
 
 class TestQuantizeEagerOps(QuantizationTestCase):
+    def _test_conv_transpose_op_impl(self, float_module_class, quantized_module_class, extra_module_kwargs, input_size):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv = float_module_class(**extra_module_kwargs)
+                self.quant = QuantStub()
+                self.dequant = DeQuantStub()
+
+            def forward(self, x):
+                x = self.quant(x)
+                x = self.dequant(x)
+                x = self.conv(x)
+                x = self.quant(x)
+                x = self.dequant(x)
+                return x
+
+        class RefM(torch.nn.Module):
```

Comment:
not sure which one you want to use quantized conv transpose (instead of reference), but the one that is using normal path will have one quant and one dequant:

```
x = self.quant(x)
x = self.convtranspose(x)
x = self.dequant(x)
```
since the quantized convtranspose takes quantized Tensor as input and outputs a quantized Tensor